suites/quincy/rbd/tier-2_rbd_regression.yaml

```diff
@@ -255,6 +263,30 @@ tests:
       name: Test abort image migration from source to destination pool
       polarion-id: CEPH-83573324
 
+  - test:
+      desc: Setting up RGW user for s3 image migration test
+      module: exec.py
+      name: Setup rgw s3 user
+      config:
+        commands:
+          - "radosgw-admin user create --uid=test_rbd --display_name='test_rbd' --access-key test_rbd --secret test_rbd"
+
+  - test:
+      name: Image migration from s3 source
+      desc: Export an image to s3 source and reinstall as new image using migration
+      module: test_rbd_migration_image_s3_object.py
+      polarion-id: CEPH-83574092
+      config:
+        ec_pool_config:
+          pool: rbd_pool
+          data_pool: rbd_ec_pool
+          image: rbd_image
+          size: 100M
+        rep_pool_config:
```

Comment:
Does it cover Replication pool also, I did not see any execution for rep_image.?
In logs attached i can see only ec pool image migration.
log link checked http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-LJHAYF/ 
if there is any other log ink please correct me.

Reply:
Marking as resolved as have received approval from Rajendra